src/styles/_themes.scss

```diff
@@ -122,9 +136,17 @@
   --paragraph-text: #{$theme-white};
 
   // Layout
+  --bg: #{linear-gradient(
+      180deg,
+      #000000 6.36%,
+      #0f305a 73.55%,
+      #555f92 111.89%
+    )};
+
   --body-bg: #{$theme-spacebase-dark};
   --bg: #{$theme-spacebase-dark};
   --text: #{$white};
+  --header-image: none;
```

Comment:
thought: this looks strange to me, but confirmed it is what is in figma. Probably waiting for a dark version of the header image.